### DISCUSSION TITLE
Overlay ID where?

### DISCUSSION THREAD

**Discussion by Megaddd** [adamk33n3r/runelite-watchdog#173] [Q&A] [answered]
It is not entirely clear where the notification option of 'Dismiss Overlay' gets the ID from, when creating a 'sticky' Overlay notification from another notification.

**Comment by adamk33n3r** [comment, marked as answer]
You make it up yourself. You type some ID into the box when you make an overlay sticky, and put that same ID into the Dismiss Overlay box

**Reply by Megaddd** [reply]
See I tried that, but, at least with the 'test whole alert' option, it doesn't seem to dismiss
![image](https://github.com/user-attachments/assets/4930e335-27db-460a-974c-f49019521657)
![image](https://github.com/user-attachments/assets/a1512fc3-936e-486b-9888-5e9f4ca64729)
![image](https://github.com/user-attachments/assets/2834b600-bf49-4e7d-8792-5725fb9cf519)

**Reply by Megaddd** [reply]
I might be stunted. That is a screen marker, not an overlay.

**Reply by adamk33n3r** [reply]
Ya, that is a screen marker 😂 I should probably add a new type or option to work with screen markers. That sounds useful.

**Reply by adamk33n3r** [reply]
Added dismiss screen marker notification type here https://github.com/runelite/plugin-hub/pull/7191

**Reply by adamk33n3r** [reply]
This has been merged and should now be in the plugin!
